cpp/box/Box.h

```diff
@@ -336,12 +336,17 @@ class Box
     /*! \param vecs Vectors to wrap, updated to the minimum image obeying the periodic settings
      *  \param Nvecs Number of vectors
      */
-    void wrap(vec3<float>* vecs, unsigned int Nvecs) const
+    void wrap(vec3<float>* vecs, unsigned int Nvecs, vec3<float> *out = nullptr) const
```

Comment:
Match the convention of the other pointers (`*` on the type rather than the name).
```suggestion
    void wrap(vec3<float>* vecs, unsigned int Nvecs, vec3<float>* out = nullptr) const
```